source/development/packaging/generate-dashboard-package.rst

```diff
@@ -40,17 +40,21 @@ To build the packages, follow these steps:
       # git clone -b <BRANCH_OR_TAG> https://github.com/wazuh/wazuh-dashboard.git
       # cd wazuh-dashboard/
       # yarn osd bootstrap
-      # yarn build --linux --skip-os-packages --release
+      # yarn build < --linux || --linux-arm > --skip-os-packages --release
 
    Example:
 
    .. code:: console
 
-      # git clone -b 4.9.0 https://github.com/wazuh/wazuh-dashboard.git
+      # git clone -b 4.10.2 https://github.com/wazuh/wazuh-dashboard.git
       # cd wazuh-dashboard/
       # yarn osd bootstrap
       # yarn build --linux --skip-os-packages --release
 
+   .. note::
+
+      If you want to build a package to ARM architecture, you must use the ``--linux-arm`` flag in the ``yarn build`` command instead of the ``--linux`` flag.
```

Comment:
```suggestion
      To build a package for ARM architecture, use the ``--linux-arm`` flag with the ``yarn build`` command, rather than the ``--linux`` flag.
```